support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummarySelector.js

```diff
@@ -19,19 +21,34 @@ const getPromotion = (productPrice: ProductPrice): Option<?string> =>
     : null);
 
 function mapStateToProps(state: CheckoutState): EndSummaryProps {
-  const { billingPeriod, productPrices } = state.page.checkout;
-
-  const productPrice = getProductPrice(
-    productPrices,
-    state.common.internationalisation.countryId,
-    billingPeriod,
-  );
+  const {
+    billingPeriod, productPrices, fulfilmentOption, productOption, product,
+  } = state.page.checkout;
+
+  const productPrice = (product === 'Paper') ?
+    getPaperProductPrice(
+      productPrices,
+      fulfilmentOption,
```

Comment:
You could pass fulfilmentOption and productOption into `getProductPrice` to avoid this ternary